src/PhpImap/Imap.php

```diff
@@ -911,10 +911,10 @@ public static function sort(
         \imap_errors(); // flush errors
 
         $imap_stream = self::EnsureConnection($imap_stream, __METHOD__, 1);
-        $reverse = $reverse;
 
-        /** @var int */
-        $criteria = $criteria;
```

Comment:
These two lines are required to avoid these `psalm` errors:
```
ERROR: MixedArgument - src/PhpImap/Imap.php:922:17 - Argument 2 of imap_sort cannot be mixed, expecting int (see https://psalm.dev/030)
                $criteria,


ERROR: MixedArgument - src/PhpImap/Imap.php:931:17 - Argument 2 of imap_sort cannot be mixed, expecting int (see https://psalm.dev/030)
                $criteria,


ERROR: MixedArgument - src/PhpImap/Imap.php:939:17 - Argument 2 of imap_sort cannot be mixed, expecting int (see https://psalm.dev/030)
                $criteria,
```